packages/foundry/contracts/modules/RepayCloseModule.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity >=0.8.13;
+
+import "../LadleStorage.sol";
+import "@yield-protocol/utils-v2/contracts/cast/CastU256I128.sol";
+import "@yield-protocol/utils-v2/contracts/cast/CastU256U128.sol";
+import "@yield-protocol/utils-v2/contracts/token/TransferHelper.sol";
+
+contract RepayCloseModule is LadleStorage {
+    using CastU256I128 for uint256;
+    using CastU256U128 for uint256;
+    using TransferHelper for IERC20;
+
+    ///@notice Creates a Healer module for the Ladle
+    ///@param cauldron_ address of the Cauldron
+    ///@param weth_ address of WETH
+    constructor(ICauldron cauldron_, IWETH9 weth_) LadleStorage(cauldron_, weth_) {}
+
+    /// @dev Use fyToken in the Ladle to repay debt. Return unused fyToken to `to`.
```

Comment:
"Return collateral to foo and unused fyToken to bar"

Reply:
Of course. Don't know why I neglected that part.